auth/oidc.go

```diff
@@ -261,7 +262,10 @@ func GetOIDCUsername(provider *OIDCProvider, subject string) string {
 
 // Converts an OpenID Connect / OAuth2 error to an HTTP error
 func OIDCToHTTPError(err error) error {
-	if oauthErr, ok := err.(*oauth2.Error); ok {
+
+	unwrappedErr := pkgerrors.Cause(err)
+
+	if oauthErr, ok := unwrappedErr.(*oauth2.Error); ok {
```

Comment:
I feel like doing this on a single line would make the code feel more readable.   e.g.
if oauthErr, ok := pkgerrors.Cause(err).(*oauth2.Error); ok {

I'm open to counter-arguments, but it feels a bit verbose as-is when there's only a single usage of unwrappedErr.

Reply:
Made that change, since it's probably easier to follow .. but still not ideal since `pkgerrors.Cause(err).(*oauth2.Error)` is a bit confusing looking.